### ISSUE TITLE
Localization (l10n)

### ISSUE THREAD

**Issue by jdorfman** [MaxCDN/bootstrapcdn#1176]
While checking the analytics I noticed some using Google Translate. 

![image](https://user-images.githubusercontent.com/398230/42126356-02faac04-7c3c-11e8-888a-215b3af5a411.png)

Core Value no. `3` is accessibility so I think we should be actively looking for contributors who can help translate the site to `X` language.

Here are top languages via Google Analytics

![image](https://user-images.githubusercontent.com/398230/42126399-f5928aae-7c3c-11e8-96b5-c7f89f715c00.png)

Looks like we should start with `es-es` aka Spanish.
![image](https://user-images.githubusercontent.com/398230/42126412-2db344b4-7c3d-11e8-9140-0ec7a293485f.png)

http://www.lingoes.net/en/translator/langcode.htm

@XhmikosR how will we be able to achieve this?

**Comment by XhmikosR** [comment]
No idea to be honest. Also, I'm personally against it. Our target audience is very specific. And you just can't be a developer without basic English knowledge.

**Comment by jdorfman** [comment]
You are correct. I installed [this](https://chrome.google.com/webstore/detail/advanced-page-language-sw/mhgjlbolkmcookadjpolimbddngjgbol?hl=en) and tried GitHub and SO and they both didn't redirect to the language (tried `DE`) while Stickermule.com did.
